pkg/admissioncontroller/webhook/admission/resourcesize/handler.go

```diff
@@ -100,6 +105,20 @@ func (h *Handler) handle(req admission.Request) error {
 	return nil
 }
 
+func relevantObjectSize(rawObject []byte) (int64, error) {
+	var obj map[string]any
+	err := json.Unmarshal(rawObject, &obj)
+	if err != nil {
+		return 0, err
+	}
+	delete(obj, "status")
+	if obj["metadata"] != nil {
```

Comment:
wdyt? Is this nil check required, as every persisted resource should have `ObjectMeta`, which cannot be `nil`, correct?